### DISCUSSION TITLE
Функция linkToTable

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#330] [Как это сделать?] [answered]
Как с помощью linkToTable возможно открыть таблицу у пользователя c префильтром "все кроме определенного значения"?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Чтоб по красоте – это в PRO будут «режимы работы Префильтров» (когда, не знаю).

Чтоб костыльно сейчас — это можно сделать для ограниченного количества значений.

Что делаете:

1. Передаете Префильтр с исключаемым значением
2. Этот Префильтр не привязан к строчной части
3. Делаете технический Префильтр тип Данные
4. Привязываете к строчной части именно Технический
5. Рассчитываете кодом все значения, которые должны быть видны в таблице (берете все и вычитаете исключаемое)

— из-за 5 пункта, понятно, что когда у вас там будет дофига значений все это ляжет.
